### DISCUSSION TITLE
Module Development - Edit modal

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2281] [Q&A] [answered]
Can Edit be opened as a page and not as a modal window?

**Comment by sbwalker** [comment, marked as answer]
Yes. There are a number of ways to do this...

1. Set the UseAdminContainer property in your razor file to False. This will suppress the modal and render the component using the module's container rather than the "admin" container. This option can only be set by module developers and is not configurable through the UI.
2.  In the Site Settings there is a Default Admin Container which can be set to any container you want to use. This option can be set by administrators and is configurable through the UI.

**Reply by leigh-pointer** [reply]
@sbwalker thanks. I will go for option 1.
